ftplugin/python/slime.vim

```diff
@@ -5,7 +5,7 @@ end
 
 function! _EscapeText_python(text)
   if exists('g:slime_python_ipython') && len(split(a:text,"\n")) > 1
-    return ["%cpaste -q\n", g:slime_dispatch_ipython_pause, a:text, "--\n"]
+    return ["%cpaste -q\n", g:slime_dispatch_ipython_pause, a:text, "\n--\n"]
```

Comment:
I'm really hesitant about changing the python ftplugin here — especially if this only came up with wezterm